spec/services/hyrax/work_uploads_handler_spec.rb

```diff
@@ -71,15 +71,6 @@
                               have_attributes(payload: include(file_set: be_file_set)))
       end
 
-      it 'publishes object.metadata.updated event for work' do
-        expect { service.attach }
-          .to change { listener.object_metadata_updated }
-          .to contain_exactly(have_attributes(payload: include(object: be_work)),
```

Comment:
i think we still want to test this `publish` for the work, just not the file sets?